packages/web/config/asset-list/__tests__/asset-lists.spec.ts

```diff
@@ -795,11 +879,47 @@ describe("getKeplrCompatibleChain", () => {
             "coinGeckoId": undefined,
             "coinImageUrl": "/tokens/generated/dgl.png",
             "coinMinimalDenom": "factory/juno1u805lv20qc6jy7c3ttre7nct6uyl20pfky5r7e/DGL",
-            "contractAddress": "juno1u805lv20qc6jy7c3ttre7nct6uyl20pfky5r7e",
+            "contractAddress": undefined,
```

Comment:
Is this expected? I see the contact address in the coinMinimalDenom.

Reply:
See slack, but I don't believe it is needed